photo-blocks/block-template-parts/footer.html

```diff
@@ -0,0 +1,11 @@
+<!-- wp:spacer {"height":38} -->
```

Comment:
There's a lot of blocks with this exact setting - could we make it a reusable block, or a block style?

Reply:
Hmm that's a good point! A block style might get confusing if the user were to edit it... and themes can't bundle reusable blocks today... I _think_ the right way to do this would be to have a global styles setting for the default spacer block height. I'll open a Gutenberg issue for it.

Reply:
Issue here: https://github.com/WordPress/gutenberg/issues/27614